deploy/application/python/tutorials/python-django-sample.md

```diff
@@ -23,6 +23,10 @@ The application is a very basic one. More information about the application:
 
 ## Configure your Django application
 
+Note : 
+- Select at least a Nano instance (Pico doesn't have enough resources for a Django project).
+- Connect your Django project to Postgresql version 12 if you're using django 4.2+ (postgresql 11 is deprecated since this version).
```

Comment:
```suggestion
Note:
 
- Select at least a `nano` scaler (`pico` doesn't have enough resources for a Django project).
- Connect your Django project to PostgreSQL version 12 if you're using Django 4.2+ (PostgreSQL 11 is deprecated since this version).
```